radosgw-agent/build/build

```diff
@@ -23,21 +44,22 @@ if [[ -f /etc/redhat-release || -f /usr/bin/zypper ]] ; then
             python setup.py bdist_rpm
             if [ $? -eq 0 ]
             then
-                mv dist/*.noarch.rpm $WORKSPACE/dist/noarch/.
-                mv dist/*.src.rpm $WORKSPACE/dist/SRPMS
+                find dist -name "*.noarch.rpm" | $VENV/chacractl binary ${chacra_flags} create ${chacra_endpoint}/noarch
+                find dist -name "*.src.rpm" | $VENV/chacractl binary ${chacra_flags} create ${chacra_endpoint}/source
             fi
         else
+            mkdir -p $HOME/rpmbuild/{BUILD,BUILDROOT,RPMS,SOURCES,SPECS,SRPMS}
+
             python setup.py clean
             python setup.py sdist --formats=gztar
-            rpmdev-setuptree
-            rpmdev-wipetree
```

Comment:
I don't feel good about this final change here; I'd rather we just install rpmdevtools on all the CentOS slaves